crates/bevy_winit/src/winit_windows.rs

```diff
@@ -10,6 +13,7 @@ pub struct WinitWindows {
 }
 
 impl WinitWindows {
+    #[allow(clippy::float_cmp)]
```

Comment:
With my other changes this should no longer be needed

Reply:
But there is: 'if max_width < min_width" and "if max_height < min_height"

Reply:
Oh true, yeah

I think it's fine, although what does clippy want us to do instead?